--- origsrc/jhbuild-2.28.0/jhbuild/frontends/gtkui.py	2009-08-19 10:15:19.000000000 -0500
+++ src/jhbuild-2.28.0/jhbuild/frontends/gtkui.py	2009-11-02 21:32:05.306888300 -0600
@@ -64,14 +64,7 @@ class AppWindow(gtk.Window, buildscript.
         gtk.Window.__init__(self)
         self.set_resizable(False)
         theme = gtk.icon_theme_get_default()
-        gtk.window_set_default_icon_list(
-                theme.load_icon('applications-development', 16, ()),
-                theme.load_icon('applications-development', 24, ()),
-                theme.load_icon('applications-development', 32, ()),
-                theme.load_icon('applications-development', 48, ()),
-                theme.load_icon('applications-development', 64, ()),
-                theme.load_icon('applications-development', 128, ())
-                )
+        gtk.window_set_default_icon_name('applications-development')
         self.set_title('JHBuild')
 
         self.module_set = jhbuild.moduleset.load(config)
